response_test.go

```diff
@@ -85,6 +85,18 @@ func TestResponseFailed(t *testing.T) {
 	})
 }
 
+func TestMultipleRttArgs(t *testing.T) {
+	check := func(resp *Response) {
+		resp.chain.assertFailed(t)
+	}
```

Comment:
I think this check() function is unnecessary, we can inline it.

Reply:
Ok sure @gavv